src/lib.rs

```diff
@@ -90,7 +96,7 @@ pub enum Command {
     },
     ControllerUpdate(Option<ControllerUpdate>, Option<ControllerUpdate>),
     USD {
-        root: Option<PathBuf>,
+        root: Option<String>,
```

Comment:
Maybe look into the unix_path crate as I think that would be a better option that a plain string